### PR TITLE
Update options tab and lookup box

### PR DESCRIPTION
**Pull request by irina060981**
For the issue https://github.com/alpheios-project/alpheios-core/issues/104

- create feature options - `showBetaCodesInfo` and `useBetaCodes`
- move options `enableLogeionAutoComplete`,  `showBetaCodesInfo` and `useBetaCodes` to Options/Features tab

image

For the issue https://github.com/alpheios-project/alpheios-core/issues/309

- create new Options tab - Advanced
- place there the following options from UI tab - `verboseMode`, `mouseMoveDelay`, `mouseMoveAccuracy`, `enableMouseMoveLimitedByIdCheck`, `mouseMoveLimitedById`

image

@balmas, I decided that creation of additional type of settings - advanced settings would be redundant, so this grouping to Advanced tab is only visual

We still have ui, feature and reasource settings

### CODE REVIEW

File: packages/components/src/settings/feature-options-defaults.json [hunk 1]
```diff
@@ -213,6 +213,36 @@
...
+        },
+        {
+          "value": false,
+          "text": "No"
+        }
+      ]
+    },
+    "useBetaCodes": {
+      "defaultValue": true,
+      "labelText": "Use Betacodes for typing greek inside Lookup box",
```
Comment (balmas) [inline packages/components/src/settings/feature-options-defaults.json:234]: change labelText to "Convert Greek Betacode to Unicode"

File: packages/components/src/settings/feature-options-defaults.json [hunk 2]
```diff
@@ -213,6 +213,36 @@
       "minValue": 1,
       "maxValue": 100,
       "values": []
+    },
+    "showBetaCodesInfo": {
+      "defaultValue": false,
+      "labelText": "Show Betacode to Unicode Mappings inside Lookup box",
```
Comment (balmas) [inline packages/components/src/settings/feature-options-defaults.json:219]: change labelText to "Show Betacode to Unicode Mapping Help"

File: packages/components/src/settings/feature-options-defaults.json (hunk 1 above)
Comment (irina060981) [reply packages/components/src/settings/feature-options-defaults.json:234]: done

File: packages/components/src/settings/feature-options-defaults.json (hunk 2 above)
Comment (irina060981) [reply packages/components/src/settings/feature-options-defaults.json:219]: done

**Pull request merged**